### PR TITLE
Translate to Brazilian Portuguese the operators section of the basic training

### PR DESCRIPTION
**Pull request by jvfe**
- Update mkdocs config
- Create pt version of the operators section

### CODE REVIEW

**Comment by mribeirodantas** [comment]
Hello, @jvfe ! Thanks for starting to work on that 🥳

**Comment by jvfe** [comment]
Can't add you as a reviewer @mribeirodantas so I'm tagging you 😶‍🌫️

File: docs/basic_training/operators.pt.md [hunk 1]
```diff
@@ -0,0 +1,271 @@
...
+
+Existem sete grupos de operadores descritos em detalhe na Documentação do Nextflow, estes são:
+
+1. [Operadores de filtragem](https://www.nextflow.io/docs/latest/operator.html#filtering-operators)
+2. [Operadores de transformação](https://www.nextflow.io/docs/latest/operator.html#transforming-operators)
+3. [Operadores de divisão](https://www.nextflow.io/docs/latest/operator.html#splitting-operators)
+4. [Operadores de combinação](https://www.nextflow.io/docs/latest/operator.html#combining-operators)
+5. [Operadores de bifurcação](https://www.nextflow.io/docs/latest/operator.html#forking-operators)
+6. [Operadores matemáticos](https://www.nextflow.io/docs/latest/operator.html#maths-operators)
+7. [Outros operadores](https://www.nextflow.io/docs/latest/operator.html#other-operators)
```
Comment (jvfe) [inline docs/basic_training/operators.pt.md:17]: Is this accurate? The sections in the website seem to be different (e.g. Reduction, Parsing Text Data)

Comment (mribeirodantas) [reply docs/basic_training/operators.pt.md:17]: Good point. It's outdated, but we can't do anything from the translation side. I will fix it on the original document, but before we must set up a system that will automatically flag outdated translations. I will keep this one to test the system once it's set up.

File: docs/basic_training/operators.pt.md [hunk 2]
```diff
@@ -0,0 +1,271 @@
...
+    .view()
+```
+
+Um `map` pode associar uma _tupla_ genérica a cada elemento e pode conter qualquer
+tipo de dado.
+
+```groovy linenums="1"
+Channel
+    .of( 'olá', 'mundo' )
+    .map { word -> [word, word.size()] }
```
Comment (mribeirodantas) [inline docs/basic_training/operators.pt.md:101]:
```suggestion
    .map { palavra -> [palavra, palavra.size()] }
```

If the English words were translated in the other examples, I think we should do it everywhere.

File: docs/basic_training/operators.pt.md [hunk 3]
```diff
@@ -0,0 +1,271 @@
...
+    .view { word, len -> "$word contém $len letras" }
```
Comment (mribeirodantas) [inline docs/basic_training/operators.pt.md:102]:
```suggestion
    .view { palavra, comprimento -> "$palavra contém $comprimento letras" }
```

File: docs/basic_training/operators.pt.md [hunk 4]
```diff
@@ -0,0 +1,271 @@
...
+!!! exercise
+
+    Use `fromPath` para criar um canal emitindo os arquivos _fastq_ que correspondam à expressão `data/ggal/*.fq`, então use `map` para retornar um par contendo o nome e o caminho para o arquivo, e, por fim, use `view` para imprimir o canal resultante.
+
+    ??? solution
+
+        ```groovy linenums="1"
+        Channel
+            .fromPath('data/ggal/*.fq')
+            .map { file -> [ file.name, file ] }
```
Comment (mribeirodantas) [inline docs/basic_training/operators.pt.md:114]:
```suggestion
            .map { arquivo -> [ arquivo.name, arquivo ] }
```

File: docs/basic_training/operators.pt.md [hunk 5]
```diff
@@ -0,0 +1,271 @@
...
+            .view { name, file -> "> $name : $file" }
```
Comment (mribeirodantas) [inline docs/basic_training/operators.pt.md:115]:
```suggestion
            .view { nome, arquivo -> "> $nome : $arquivo" }
```

File: docs/basic_training/operators.pt.md [hunk 6]
```diff
@@ -0,0 +1,271 @@
...
+
+!!! exercise
+
+    Use `fromPath` para criar um canal emitindo todos os arquivos no diretório `data/meta/`, então use `map` para associar o prefixo `baseName` a cada arquivo. Por fim, agrupe todo os arquivos que possuem o mesmo prefixo.
+
+    ??? solution
+
+        ```groovy linenums="1"
+        Channel.fromPath('data/meta/*')
+            .map { file -> tuple(file.baseName, file) }
```
Comment (mribeirodantas) [inline docs/basic_training/operators.pt.md:216]:
```suggestion
            .map { arquivo -> tuple(arquivo.baseName, arquivo) }
```

File: docs/basic_training/operators.pt.md [hunk 7]
```diff
@@ -0,0 +1,271 @@
...
+            .groupTuple()
+            .view { baseName, file -> "> $baseName : $file" }
```
Comment (mribeirodantas) [inline docs/basic_training/operators.pt.md:218]:
```suggestion
            .view { baseName, arquivo -> "> $baseName : $arquivo" }
```

File: docs/basic_training/operators.pt.md [hunk 8]
```diff
@@ -0,0 +1,271 @@
...
+[X, 1, 4]
+```
+
+!!! note
+
+    Perceba como _P_ está ausente no resultado final.
+
+### `branch()`
+
+O operador `branch` permite que você envie os itens emitidos por um canal de entrada para um ou mais canais de saída.
```
Comment (mribeirodantas) [inline docs/basic_training/operators.pt.md:244]: Aqui você traduz `source channel` como canal de entrada. No entanto, mais cedo nesta seção, você traduziu como canal original. Vamos tentar escolher um nome, e mantê-lo ao longo do material. Qual prefere?

**Review by mribeirodantas — CHANGES_REQUESTED** [review]

File: docs/basic_training/operators.pt.md (hunk 1 above)
Comment (jvfe) [reply docs/basic_training/operators.pt.md:17]: Agreed!

File: docs/basic_training/operators.pt.md (hunk 8 above)
Comment (jvfe) [reply docs/basic_training/operators.pt.md:244]: Acredito que canal de entrada fica mais claro de entender, afinal canal original pode ser muita coisa. Vou mudar a outra ocorrência.

**Review by mribeirodantas — APPROVED** [review]

**Comment by mribeirodantas** [comment]
Great work, @jvfe ! 🥳

**Pull request merged**